### PR TITLE
[DOCFIX] Add runHdfsMountTests and runUfsIOTest to UserCLI doc

### PR DESCRIPTION
**Pull request by jiacheliu3**

### CODE REVIEW

File: docs/en/operation/User-CLI.md [hunk 1]
```diff
@@ -251,11 +251,11 @@ Note that this command will stop any Alluxio services running on the machine.
...
-`-h` provides detailed guidance.
-`-m <hive_metastore_uris>` (required) the full hive metastore uris to connect to an existing hive metastore.
-`-d <databse_name>` the database to run tests against. Use `default` database if not provided.
-`-t` tables to run tests against. Run tests against five out of all tables in the given database if not provided.
-`-st` socket timeout of hive metastore client in minutes.
+* `-h` provides detailed guidance.
+* `-m <hive_metastore_uris>` (required) the full hive metastore uris to connect to an existing hive metastore.
+* `-d <databse_name>` the database to run tests against. Use `default` database if not provided.
+* `-t` tables to run tests against. Run tests against five out of all tables in the given database if not provided.
+* `-st` socket timeout of hive metastore client in minutes.
```
Comment (jiacheliu3) [inline docs/en/operation/User-CLI.md:258]: W/o the bulletpoints the lines will cramp together.

**Comment by madanadit** [comment]
alluxio-bot, merge this please.

**Comment by madanadit** [comment]
alluxio-bot, cherry-pick this to enterprise-2.3 please.

**Comment by madanadit** [comment]
alluxio-bot, cherry-pick this to enterprise-2.3 please.

**Comment by madanadit** [comment]
alluxio-bot, merge this to branch-2.3 please.

**Comment by madanadit** [comment]
alluxio-bot, cherry-pick this to branch-2.3 please.

**Pull request merged**